### PR TITLE
CATTY-472 UI changes for store details

### PR DESCRIPTION
**Pull request by Bortol15**
Added changes to the detail store according to the Ticket: https://jira.catrob.at/browse/CATTY-472

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Verify that the Jira ticket is in the status *Ready for Development*
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by Bortol15** [comment]
jenkins retest

**Comment by Bortol15** [comment]
Jenkins retest this

**Comment by Bortol15** [comment]
test this please

**Comment by Bortol15** [comment]
test this please

**Review by srinner — APPROVED** [review]

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift [hunk 1]
```diff
@@ -161,240 +228,170 @@ import ActiveLabel
...
+        let views = project.views
+        let downloads = project.downloads
+        var size = project.size
+
+        let offsetLine = descriptionView.frame.origin.y + descriptionView.frame.size.height + type(of: self).verticalSectionPadding
+        let offsetLabel = offsetLine + type(of: self).verticalSectionPadding
+
+        self.addHorizontalLine(to: view, verticalOffset: offsetLine)
+
+            let informationLabel = UILabel(frame: CGRect(x: type(of: self).inset, y: offsetLabel, width: view.frame.width, height: type(of: self).labelHeight))
```
Comment (wallisch) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:304]: Just a small format error

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift [hunk 2]
```diff
@@ -20,29 +20,66 @@
...
         view.backgroundColor = UIColor.clear
-        view.autoresizingMask = .flexibleHeight
-        self.addNameLabel(withProjectName: project.projectName, to: view)
-        self.addAuthorLabel(withAuthor: project.author, to: view)
-        self.addThumbnailImage(withImageUrlString: project.screenshotSmall, to: view)
-        self.addDownloadButton(to: view, withTarget: target)
-        self.addLoadingButton(to: view, withTarget: target)
-        self.addOpenButton(to: view, withTarget: target)
+
+        let thumbnailView = self.addThumbnailImage(withImageUrlString: project.screenshotSmall, to: view)
```
Comment (wallisch) [inline src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:67]: Maybe we should use screenshotBig here, the small one looks extremely blurry on larger devices

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift (hunk 1 above)
Comment (m-herold) [reply src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift:304]: Nice catch, thanks.

File: src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift (hunk 2 above)
Comment (m-herold) [reply src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift]: Great idea, done!

**Review by wallisch — APPROVED** [review]

**Pull request merged**